extensions/api/data-management/transferprocess/src/main/java/org/eclipse/dataspaceconnector/api/datamanagement/transferprocess/TransferProcessApiController.java

```diff
@@ -133,7 +134,7 @@ public void deprovisionTransferProcess(@PathParam("id") String id) {
     @Consumes({ MediaType.APPLICATION_JSON })
     @Produces({ MediaType.TEXT_PLAIN })
     @Override
-    public String initiateTransfer(TransferRequestDto transferRequest) {
+    public String initiateTransfer(@Valid TransferRequestDto transferRequest) {
         if (!isValid(transferRequest)) {
```

Comment:
this validation could be done with the jakarta validation as well

Reply:
should be implicitly done, removed `isValid`